docs/source/chatops/chatops.rst

```diff
@@ -51,12 +51,47 @@ Configuration
 Package-based Install
 ~~~~~~~~~~~~~~~~~~~~~
 
-If you installed StackStorm following the :doc:`install docs </install/index>`, the ``st2chatops``
-package will take care of everything for you. Hubot with the necessary adapters is already bundled,
-and environment variables are sourced from ``/opt/stackstorm/chatops/st2chatops.env``.
+If you installed |st2| following the :doc:`install docs </install/index>`, the ``st2chatops``
+package will take care of `almost` everything for you. Hubot with the necessary adapters is already
+installed, and StackStorm API keys have been configured. 
 
-Edit the file to specify your Chat service and bot credentials. If you need extra environment
-settings for Hubot, you should store them in ``st2chatops.env`` as well.
+You just need to tell |st2| which Chat service to use - e.g. Slack, MatterMost, etc. You will also need
+to give it credentials. Your Chat service may also need configuration. For example, to configure Slack,
+you first need to add a new Hubot integration to Slack. You can do this through Slack's admin interface.
+Take note of the ``HUBOT_SLACK_TOKEN`` that Slack provides.
+
+Then edit the file ``/opt/stackstorm/chatops/st2chatops.env``. Edit and uncomment the variables for 
+your adapter. For example, if you are configuring Slack, look for this section:
+
+.. code-block:: bash
+
+    # Slack settings (https://github.com/slackhq/hubot-slack):
+    #
+    # export HUBOT_ADAPTER=slack
+    # Obtain the Slack token from your app page at api.slack.com, it's the "Bot
+    # User OAuth Access Token" in the "OAuth & Permissions" section.
+    # export HUBOT_SLACK_TOKEN=xoxb-CHANGE-ME-PLEASE
+    # Uncomment the following line to force hubot to exit if disconnected from slack.
+    # export HUBOT_SLACK_EXIT_ON_DISCONNECT=1
+
+Edit this file so it looks something like this:
+
+.. code-block:: bash
+
+    # Slack settings (https://github.com/slackhq/hubot-slack):
+    #
+    export HUBOT_ADAPTER=slack
+    # Obtain the Slack token from your app page at api.slack.com, it's the "Bot
+    # User OAuth Access Token" in the "OAuth & Permissions" section.
+    export HUBOT_SLACK_TOKEN=xoxb-SUPER-SECRET-TOKEN
+    # Uncomment the following line to force hubot to exit if disconnected from slack.
+    export HUBOT_SLACK_EXIT_ON_DISCONNECT=1
+
+Your specific Chat service may require different settings. Any environment settings needed can be
+added to this file. 
+
+Once you have finished making changes, restart ``st2chatops`` with ``sudo systemctl restart st2chatops``.
```

Comment:
`sudo service st2chatops restart` will do the trick better for both systemd and other init systems, covering older & new distros which we still support.

Reply:
I went back & forth on that one. You're right for now (until we can get rid of RHEL6 & U14 anyway)